tests/utils/test_get_instance.py

```diff
@@ -1,71 +1,81 @@
 """Tests oteapi-dlite.utils.get_instance()."""
-from pathlib import Path
+# pylint: disable=too-many-locals
+from typing import TYPE_CHECKING
 
-import dlite
-import numpy as np
-from tripper import EMMO, MAP, Triplestore
+if TYPE_CHECKING:
+    from pathlib import Path
 
-from oteapi_dlite.utils import get_instance
 
-# Setup
-thisdir = Path(__file__).absolute().parent
-entitydir = thisdir.parent / "entities"
-dlite.storage_path.append(f"{entitydir}/*.json")
+def test_instantiate_calcresults(entities_path: "Path") -> None:
+    """Test utils.get_instance().
```

Comment:
Same comment as for the other test. The documentation is good and reduces the obfuscation of what is going on.

Still, I prefer the script.

Reply:
We have agreed to follow the best practices of writing unit tests for our collective code within the team. This change follows that directive.